.github/ISSUE_TEMPLATE/orientation-epic.md

```diff
@@ -38,6 +38,13 @@ As a new VFS team member, I need to complete onboarding activities so I can succ
 ### General
 
 - [ ] Attend General Orientation meeting ([schedule via Calendly](https://calendly.com/collaboration-cycle/general-orientation-for-va-gov-platform?month=2021-11))
+- [ ] Schedule Role-based Orientation meeting (schedule via Calendly)
```

Comment:
@seanasewell all of the role-based changes are already included in the template but right now they say to "DM" whoever the host is. Can you remove this section and place the links in the appropriate places? I'll comment where they should be.

Reply:
ok for some reason i can't add comments. However under "### Get set up for your practice area" there are several sections that are organized by role. For example, under "Designers and Researchers" it says to "Request design orientation with @ Allison Christman in Slack." This needs to be replaced with, "Attend Design Orientation meeting (schedule via Calendly)" and link the url. Make sense?